js/bootstrap-datepicker.js

```diff
@@ -842,14 +842,17 @@
 		},
 
 		fillMonths: function(){
-      var localDate = this._utc_to_local(this.viewDate);
-			var html = '';
-			var focused;
+			var focused, node, localDate = this._utc_to_local(this.viewDate);
+			this.picker.find('.datepicker-months td').html('');
 			for (var i = 0; i < 12; i++){
 				focused = localDate && localDate.getMonth() === i ? ' focused' : '';
-				html += '<span class="month' + focused + '">' + dates[this.o.language].monthsShort[i] + '</span>';
+				node = $('<span>');
+				node.data('date', {day: 1, month: i, year: this.viewDate.getUTCFullYear()});
+				node.attr('class', 'month' + focused);
+				node.html(dates[this.o.language].monthsShort[i]);
+				node.on('click', $.proxy(this.cellClick, this));
```

Comment:
Ok that explains a lot! So we should go for the 90 events for now? What do you think @vsn4ik?